ComparatorTask/Program.cs

```diff
@@ -1,15 +1,46 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace ComparatorTask
 {
     class Program
     {
         static void Main(string[] args)
         {
+            List<Movie> movieList = new List<Movie>();
+            movieList.Add(new Movie("First movie", 2000, 5.5));
+            movieList.Add(new Movie("The best movie", 2011, 8.7));
+            movieList.Add(new Movie("The best movie", 2008, 8.0));
+            movieList.Add(new Movie("Second movie", 2001, 4.8));
+            movieList.Add(new Movie("New movie", 2015, 5.5));
+            movieList.Add(new Movie("Movie 1", 2011, 6.5));
+            movieList.Add(new Movie("Movie 2", 2012, 7.5));
+
+           
+            movieList.Sort();
+            Console.WriteLine("--- Movies sorted by score (default) ---");
```

Comment:
add case with Sort(new MovieScoreComparator()) and compare with movieList.Sort();